config/default/webhook/manifests.yaml

```diff
@@ -31,10 +31,10 @@ webhooks:
         path: /mutate-pods
     failurePolicy: Fail
     name: inferenceservice.kfserving-webhook-server.pod-mutator
-    namespaceSelector:
+    objectSelector:
```

Comment:
This is a separate feature! Do we want to do this as part of this change??

Reply:
sure, can raise a separate PR

Reply:
reverted back